test/component_test.go

```
@@ -0,0 +1,93 @@
+	"strings"
+	"testing"
+
+	"github.com/aws/aws-sdk-go-v2/service/ec2"
+	"github.com/aws/aws-sdk-go-v2/service/ec2/types"
+	"github.com/cloudposse/test-helpers/pkg/atmos"
+	helper "github.com/cloudposse/test-helpers/pkg/atmos/aws-component-helper"
+	"github.com/gruntwork-io/terratest/modules/aws"
+	"github.com/stretchr/testify/assert"
+)
+
+type validationOption struct {
```

golangci-lint (1.62.2)

type `validationOption` is unused

(unused)